ckan/plugins/toolkit.py

```diff
@@ -138,6 +140,20 @@ def _initialize(self):
 current request.
 
 '''
+        t['config'] = pylons.config
+        self.docstring_overrides['config'] = '''The Pylons configuration object.
+
+The configuration object containing information about this request as well as
+options loaded from the .ini file.
+'''
+
+        t['translator'] = pylons.translator
+        self.docstring_overrides['translator'] = '''The Pylons translator object.
```

Comment:
What does translator give that we don't get with `_` and `ungettext` already present?